news_items/034_matching_element/index.html

```diff
@@ -0,0 +1,65 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     width: 60%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+ .code-block div {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: -1em;
+     line-height: 125%;
+ }
+</style>
+
+<p>
+    PrairieLearn now has a new <code class="user-output">pl-matching</code> element, thanks to the efforts of 
+    <a href="https://github.com/nealterrell">Neal Terrell</a> (California State University, Long Beach), with input from
+    <a href="https://github.com/jonatanschroeder">Jonatan Schroeder</a>.
```

Comment:
Does @jonatanschroeder want an institutional association?

Reply:
I removed my name since I'm the author. No need to mention me, I'm getting enough attention as is 😉